tests/dataplaneapi.recipe

```diff
@@ -0,0 +1,33 @@
+# Bibop recipe for dataplaneapi
+# See more: https://kaos.sh/bibop
+
+pkg dataplaneapi
+
+require-root yes
+unsafe-actions yes
+
+var service_name dataplaneapi
+var sysconfig /etc/sysconfig/dataplaneapi
+var logrotate_config /etc/logrotate.d/dataplaneapi
+var delay 3
+
+command "-" "Check environment"
+  app dataplaneapi
+
+  service-present {service_name}
+
+  exist {sysconfig}
+  mode {sysconfig} 644
+  exist {logrotate_config}
+  mode {logrotate_config} 644
+
+command "systemctl start {service_name}" "Start dataplaneapi daemon"
```

Comment:
If version specific tools are used for the test (_like systemd in this case, which available on CentOS/RHEL >= 7_), you should add the version to file suffix. The proper name for the file will be `dataplaneapi-c7.recipe`